chart/kubeapps/templates/tests/test-tiller-proxy.yaml

```diff
@@ -18,5 +18,5 @@ spec:
     command: 
     - sh
     - -c
-    - "curl -ik -H \"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)\" $TILLER_PROXY_HOST:$TILLER_PROXY_PORT/v1/releases | grep $KUBEAPPS_RELEASE"
+    - "curl -v -o /tmp/output -ik -H \"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)\" $TILLER_PROXY_HOST:$TILLER_PROXY_PORT/v1/releases && cat /tmp/output && cat /tmp/output | grep $KUBEAPPS_RELEASE"
```

Comment:
I guess these clusters are temporary, but here we may expose sensitive information to the CI logs

Reply:
I was going to say no because this is meant for the local cluster using `kind` but the same is executed for GKE and showing the headers would show a valid token for the cluster. Good catch! I am removing the `-v`